Benchmarks/Benchmarks/TokenLifecycle/TokenLifecycle.swift

```diff
@@ -0,0 +1,139 @@
+import Benchmark
+import Foundation
+import JWTKit
+
+let customThresholds = BenchmarkThresholds(
+    relative: [.p25: 15.0, .p50: 15.0, .p75: 15.0, .p90: 15.0, .p99: 15.0],
+    absolute: [:]
```

Comment:
I'm not 100% sure but i think declaring `relative`, you still need to declare `absolute` as well?
Unless you've tried this before and know that it'll work.

Reply:
Actually IIRC this was done like this because we were running them on GH runners which were flaky and didn't work. It probably makes sense to use the defaults now